src/lib/index.js

```diff
@@ -453,7 +453,7 @@ lib.addStyleRule = function(selector, styleString) {
 
 lib.getTranslate = function(element) {
 
-    var re = /.*\btranslate\((\d*\.?\d*)[^\d]*(\d*\.?\d*)[^\d].*/,
+    var re = /.*\btranslate\((-?\d*\.?\d*)[^-\d]*(-?\d*\.?\d*)[^\d].*/,
```

Comment:
Nice catch :microscope: 

Would you mind adding a test case in [`drawing_test.js`](https://github.com/plotly/plotly.js/blob/dc7621760c2f264ab075ff285ac8e72901939d03/test/jasmine/tests/drawing_test.js#L72) to :lock: this down?

(No rush on this by the way. If you don't have the time to do this by Monday night, I'll merge your PR as is and open up an issue about this).